mantle/cmd/kola/kola.go

```diff
@@ -640,6 +640,7 @@ func getParentFcosBuildBase(stream string) (string, error) {
 		n := len(index.Releases)
 		if n == 0 {
 			// hmmm, no builds; likely a new stream. let's just fallback on testing-devel.
+			//this also applies to bump-lockfile, see https://github.com/coreos/coreos-assembler/issues/3017
```

Comment:
```suggestion
```

I don't think we need to add this comment here. I know Jonathan said the comment needed updating, but I'm not sure. In the case there are 0 builds in the release index then there are no builds. I think he was just saying that we need to update the comments in this code to be accurate.. so we'll add comments to the block of code below we are adding.

Reply:
i.e. let's update:

```
// ok, we're probably operating on a local dev build since the pipeline
// always injects the parent; just instead fetch the release index     
// for that stream and get the last build id from there                
```

To something like:

```
// ok, we're probably operating on a local dev build or in the
// bump-lockfile job since the pipeline always injects the
// parent; just instead fetch the release index for that stream
// and get the last build id from there                
```

Reply:
if the recent comments added does not help to this, I can gladly change it. could you please have a look? 
I lost your comment inbetween github refresh(s)

Reply:
let's still update the comment above at the top of the `else`. A suggestion of what it could be is in  https://github.com/coreos/coreos-assembler/pull/3080#discussion_r965051995